package.json

```diff
@@ -64,35 +64,35 @@
   },
   "devDependencies": {
     "@babel/cli": "^7.19.3",
-    "@babel/core": "^7.20.2",
+    "@babel/core": "^7.20.5",
     "@babel/preset-env": "^7.20.2",
-    "@rollup/plugin-babel": "^6.0.2",
-    "@rollup/plugin-commonjs": "^23.0.2",
+    "@rollup/plugin-babel": "^6.0.3",
+    "@rollup/plugin-commonjs": "^24.0.0",
```

Comment:
I don't think this plugin is actually used in our rollup config.  Looks like twbs/bootstrap only uses it in tests (as far as I can tell).
```suggestion
```

Reply:
FWIW, I think it was me who originally added the rollup plugins (way back in https://github.com/az-digital/arizona-bootstrap/pull/108) and IIRC, I just included all `rollup` packages without doing much detailed investigation about where they were used...